open_bus_stride_api/routers/common.py

```diff
@@ -7,6 +7,21 @@
 from open_bus_stride_db.db import get_session
 
 
+FILTER_DOCS = {
+    "list": 'Filter by {0}. Comma-separated list of values.',
```

Comment:
please use a named format specifier, so we will be able to extend this in the future with other values to replace and it will be clearer what is replaced there

```
"list": 'Filter by {what_singular}. Comma-separated list of values.',
```